superset-frontend/src/components/MetadataBar/MetadataBar.test.tsx

```diff
@@ -265,3 +265,27 @@ test('correctly renders the tags tooltip', async () => {
     );
   });
 });
+
+test('renders StyledItem with role="button" when onClick is defined', () => {
+  const onClick = jest.fn();
+  const items = [
+    { ...ITEMS[0], onClick },
+    { ...ITEMS[1], onClick },
+  ];
+  render(<MetadataBar items={items} />);
+
+  const styledItems = screen.getAllByRole('button');
+
+  styledItems.forEach(styledItem => {
+    expect(styledItem).toHaveAttribute('role', 'button');
+  });
```

Comment:
This seems redundant. Maybe we can check whether these elements exist by checking the length?

Reply:
Yes, this will be changed to: 
```
  const styledItems = screen.getAllByRole('button');

  expect(styledItems.length).toBe(2);
```